SMITHY_GO_CODEGEN_VERSION

```diff
@@ -1 +1 @@
-e6338ca9b5ee30c6317ea908173a369a9d3eb236
+48250f49d98ad6bed6c302a8af97f62cc7f3bd83
```

Comment:
Hold on, is this right?

[this](https://github.com/aws/smithy-go/commit/a4c9efcda6aa54c75d1a130d1320a2709eebf51d) seems to be the latest (w/ smithy upgrade)

Reply:
Great catch, will update it